src/sagas.js

```diff
@@ -802,6 +808,60 @@ function* removeVm (action) {
   yield stopProgress({ vmId: action.payload.vmId, name: 'remove', result })
 }
 
+function* fetchAllEvents (action) {
+  const { userId } = action.payload
+  const events = yield callExternalAction('events', Api.events, { payload: {} })
+
+  if (events.error) {
+    return
+  }
+
+  const internalEvents = events.event
+    ? events.event
+      .filter((event) =>
+        event.severity === 'error' &&
+        event.user &&
+        event.user.id === userId
```

Comment:
I see some event with a user id (the UID) and I see some event with just a user name (name@domain).

Probably should check both.

Reply:
Hmm... strange I didn't saw that case, I'll check.